### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by HollowPrincess**

### CODE REVIEW

**Comment by HollowPrincess** [comment]
- [ ] Merge master in the branch before merging this PR: it has critical changes that affect tests.

File: tests/all_test.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
+""" Test script for running tests notebooks with provided parameters.
+
+The behavior of all tests is controlled by the following constants that are declared in the `common_params` dict:
+
+TESTS_ROOT_DIR : str
+    Path to the directory for saving results and temporary files for all tests
+    (executed notebooks, logs, data files like cubes, etc.).
+    Note that in case of success this directory will be removed (if `REMOVE_ROOT_DIR` is True).
+REMOVE_ROOT_DIR : bool
+    Whether to remove tests root directory after execution in case of success.
```
Comment (cdtn) [inline tests/all_test.py:10]: define success

File: tests/all_test.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+SHOW_FIGURES : bool
+    Whether to show additional figures in the executed notebooks.
+    Showing some figures can be useful for finding out the reason for the failure of tests.
+VERBOSE : bool
+    Whether to print additional information from tests.
```
Comment (cdtn) [inline tests/all_test.py:15]: print where?

File: tests/all_test.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+pytest.failed = False
+BASE_DIR =  os.path.normpath(os.getenv("BASE_DIR", os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../seismiqb')))
+TESTS_NOTEBOOKS_DIR = os.path.join(BASE_DIR, 'tests/notebooks/') # path to the directory with tests notebooks
+# TUTORIALS_DIR = os.path.join(BASE_DIR, 'tutorials/') # path to the directory with tutorials
+
+# Initialize common parameters for all tests notebooks
+common_params = {
+    # Workspace constants and parameters
+    'TESTS_ROOT_DIR': os.getenv('SEISMIQB_TESTS_ROOT_DIR') or tempfile.mkdtemp(prefix='tests_root_dir_', dir=BASE_DIR),
+    'REMOVE_ROOT_DIR': os.getenv('SEISMIQB_TESTS_REMOVE_ROOT_DIR') or True,
```
Comment (cdtn) [inline tests/all_test.py:46]: always true

File: tests/all_test.py [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+    # Visualization and output parameters (these variables are used in notebooks)
+    'SHOW_FIGURES': os.getenv('SEISMIQB_TESTS_SHOW_FIGURES') or False,
+    'VERBOSE': os.getenv('SEISMIQB_TESTS_VERBOSE') or True
```
Comment (cdtn) [inline tests/all_test.py:50]: also always true

**Pull request merged**